rtv/content.py

```diff
@@ -122,7 +122,7 @@ def strip_praw_submission(sub):
         data['score'] = '{} pts'.format(sub.score)
         data['author'] = name
         data['permalink'] = sub.permalink
-        data['subreddit'] = strip_subreddit_url(sub.permalink)
+        data['subreddit'] = str(sub.subreddit)
```

Comment:
The last time I checked, calling the **sub.subreddit** property forced praw to make an extra API call to grab the subreddit object. This doubled the loading time because each call is rate limited to every 1 or 2 seconds. Can you verify that this is no longer the case?